scripts/README.md

```diff
@@ -0,0 +1,98 @@
+# scripts
```

Comment:
Having whole code don't make sense here. Any one can look up to the file and read the code.

Reply:
Please add short descriptions about the folders only, not the files as they are bound to change in logic in every PR.